### DISCUSSION TITLE
How do I make a button to edit a row in a table through an application?

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#4194] [Help] [answered]
I want to make this edit button, open up a small form to update the row you click the button on. My problem is that I have no clue how to do this. Anything I could try?

![image](https://user-images.githubusercontent.com/26911654/151027115-4253ee11-0478-4486-9d51-e3c01fa9babc.png)

**Comment by mjashanks** [comment, marked as answer]
Whenever you add a new screen, you have the option to generate screens from a table. I would recommend that you use this feature, as it generates screens
1. List rows `/your-table`
2. Edit row `/your-table/:id`
3. New row `/your-table/new/row`

The list rows screen has an edit button as you have shown above - you can click on `Define Actions` to see what it does - i.e. it just navigates to the Edit Row screen.

**Reply by adamsanclemente** [reply]
Ye I figured it out, I have another stranger issue though so I'm gonna make a new help ticket
